bounce/db/club.py

```diff
@@ -83,12 +83,21 @@ def search(session, query=None, page=0, size=MAX_SIZE):
     offset_num = page * size
     clubs = session.query(Club)
 
-    if query:
-        # show clubs that have a name that matches the query
-        clubs = clubs.filter(Club.name.ilike(f'%{query}%'))
-    else:
+    not_null_filters = []
+
+    triggered = False
+    if name or description:
+        triggered = True
+    if name:
+        not_null_filters.append(Club.name.ilike(f'%{name}%'))
+    if description:
+        not_null_filters.append(Club.description.ilike(f'%{description}%'))
+
+    if triggered is False:
```

Comment:
Minor: I don't think you need the `triggered` variable, you should be able to do
```python
if len(not_full_filters) == 0:
```

Reply:
i changed it to if not  not_null_filters: 
this is the recommended way to do it. 
using len to check if something is empty fails a lint